test/test_build.py

```diff
@@ -38,6 +38,7 @@ class ImageBuildResult(NamedTuple):
     img_arch: str
     container_ref: str
     rootfs: str
+    partition_mode: str
```

Comment:
This confused me a bit because it made me think of the [`disk.PartitioningMode`](https://github.com/osbuild/bootc-image-builder/blob/47e0292b184645d8a8d82278f67af9c7d274028a/bib/cmd/bootc-image-builder/image.go#L210-L216) that we have for the older partitioning customizations.  This is only meant for tests and not user facing, so it's not really that important, but it might be confusing for test writing and reading.

I suggest we rename it to something else, something that implies it's a keyword for a particular test partitioning configuration (which is what it is, right?).  Partitioning config maybe?

Reply:
Yeah, excellent feedback, I went with `disk_config` for now but `partion_config` or `disk_customization` would of course also be an option